common/test_local_filestore.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for local_filestore.py."""
+
+from unittest import mock
+
+import pytest
+
+from common import local_filestore
+from test_libs import utils as test_utils
+
+
+def test_local_filestore_command():
+    """Tests local_filestore_command works as expected."""
+    arguments = ['hello']
+    with test_utils.mock_popen_ctx_mgr() as mocked_popen:
+        local_filestore.local_filestore_command(arguments)
+    assert mocked_popen.commands == [arguments]
+
+
+class TestLocalUtilsRsync:
+    """Tests for local_filestore_command works as expected."""
+    SRC = '/src'
+    DST = '/dst'
+
+    @mock.patch('os.path.isdir')
+    def test_rsync_file_to_dir(self, mock_isdir):
```

Comment:
Since `gsutil rsync -r file dir` or `gsutil rsync -r file1 file2` will take no effect, shall we add the logic that `source` should be a dir (locally or cloudly) in `filestore_utils`?
For `rsync`:
- `rsync -r file dir` will put `file` under `dir`, and
- `rsync -r file1 file2` will replace `file2` content with `file1` but keep the `file2` filename.